src/styles/components/brand.css

```diff
@@ -1,3 +1,3 @@
 .michelada {
-    @apply bg-red text-white p-2 font-bold hover:bg-blue-dark;
+    @apply bg-red text-white p-2 font-bold hover:bg-purple-dark;
   }
```

Comment:
If you use VS Code, look for `Insert Final Newline` on settings. That adds a new line when you save your document.

Reply:
This will be added.